### DISCUSSION TITLE
[@types/matter-js] [Vertex]: Should index, body, isInternal be optional?

### DISCUSSION THREAD

**Discussion by lunacd** [DefinitelyTyped/DefinitelyTyped#61655] [Issues with a @types package] [answered]
The new `Vertex` interface added in v0.18.0 requires `index`, `body`, and `isInternal` in addition to `x` and `y`. Since in many cases we just want to specify a location, should those other values be optional so that TS doesn't complain about missing properties?

[a comment by an automated account is not shown]

**Comment by hasparus** [comment]
Hey @andyliuhaowen! Good catch. I overlooked it when reviewing the PR. Sorry!

**Comment by stevensnoeijen** [comment]
I'll try to fix this with a PR today :)

**Reply by hasparus** [reply]
Sorry, I didn't see your comment 😅 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61659

**Reply by stevensnoeijen** [reply]
np! well done 😁

**Comment by stevensnoeijen** [comment, marked as answer]
@andyliuhaowen  this is now fixed in v 0.18.1 https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61659, kudos to @hasparus  :)
